RiotSharpTest/RiotApiTest.cs

```diff
@@ -135,26 +135,22 @@ public void GetChampionAsync_Test()
 
         [TestMethod]
         [TestCategory("RiotApi")]
-        public void GetMasteryPages_Test()
+        public void GetMasteryPages_Summoner_HasMasteryPages()
         {
-            var fetchCount = Math.Min(10, RiotApiTestBase.summonerIds.Count());
-            var masteries = api.GetMasteryPages(RiotApiTestBase.summonersRegion, 
-                RiotApiTestBase.summonerIds.Take(fetchCount).ToList());
+            var pages = api.GetMasteryPages(RiotApiTestBase.summonersRegion, RiotApiTestBase.summoner1Id);
 
-            Assert.IsNotNull(masteries);
-            Assert.AreEqual(fetchCount, masteries.Count);
+            Assert.IsNotNull(pages);
+            Assert.AreEqual(20, pages.Count);
```

Comment:
The test will fail if the summoner deletes a mastery page. I would test for a count between 1 and 20.